tools/nvm/nvm-guide.md

```diff
@@ -0,0 +1,24 @@
+# Introduction
+
+### What is nvm?
+
+**nvm** stands for **Node Version Manager** and is a tool that allows developers to install and manage multiple versions of **Node.js** platform.
+
+# Commands
+
+### Check versions installed
+
+```sh
+nvm ls
+```
+### Install a new version
+
+```sh
+nvm install <node-version>

```

Comment:
I think this is NVM install-remote or something. I might be wrong and have no way of testing it right now. Can we plz verify it?